WcaOnRails/db/migrate/20170816143703_add_greater_china_championship_type.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddGreaterChinaChampionshipType < ActiveRecord::Migration[5.1]
+  def change
+    %w(CN HK MO TW).each do |iso2|
+      EligibleCountryIso2ForChampionship.create!(championship_type: "greater_china", eligible_country_iso2: iso2)
```

Comment:
Discussed this with @jfly before

![image](https://user-images.githubusercontent.com/17034772/29376255-40f6f112-82b8-11e7-87a1-24a2486c0140.png)

From this I understood that we plan to translate things like "Greater China" once we show champion titles. So we have to gather those championship types anyway.

I think it's hard to predict which way would be the best, especially it's difficult to estimate which will make displaying champion titles easier. At least, the current way makes the championship types UI simpler.